scripts/install_requirements_darwin_x64.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -eu
+
+INSTALL_PATH=${INSTALL_PATH:-$HOME/.edb-cloud-tools}
+
+TERRAFORM_ARCHIVE=https://releases.hashicorp.com/terraform/0.14.5/terraform_0.14.5_darwin_amd64.zip
+GCLOUD_ARCHIVE=https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-324.0.0-darwin-x86_64.tar.gz
+
+mkdir -p ${INSTALL_PATH}
+mkdir -p ${INSTALL_PATH}/bin
+chown -R ${USER} ${INSTALL_PATH}
+
+# AWS CLI
+mkdir -p ${INSTALL_PATH}/aws
+python3 -m venv ${INSTALL_PATH}/aws
+sed -i.bak 's/$1/${1:-}/' ${INSTALL_PATH}/aws/bin/activate
+source ${INSTALL_PATH}/aws/bin/activate
+pip3 install awscli
+deactivate
+ln -s ${INSTALL_PATH}/aws/bin/aws ${INSTALL_PATH}/bin/.
+
+# Azure CLI
+mkdir -p ${INSTALL_PATH}/azure
+python3 -m venv ${INSTALL_PATH}/azure
+sed -i.bak 's/$1/${1:-}/' ${INSTALL_PATH}/azure/bin/activate
+source ${INSTALL_PATH}/azure/bin/activate
+pip3 install azure-cli
+deactivate
+ln -s ${INSTALL_PATH}/azure/bin/az ${INSTALL_PATH}/bin/.
```

Comment:
Let's add -f option. So, that user can re-run the requirement script without failure.